graylog2-server/src/main/java/org/graylog2/web/customization/Config.java

```diff
@@ -17,20 +17,22 @@
 package org.graylog2.web.customization;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.Optional;
 
 
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
 public record Config(
-        Optional<String> productName,
+        @JsonProperty("product_name") Optional<String> productName,
```

Comment:
Why is this required?

Reply:
@dennisoelkers You mean `@JsonProperty("product_name")` ? That was a recommendation from  Lukasz to be more explicit.